doc/release/1.8.0-notes.rst

```diff
@@ -50,80 +86,808 @@ New features
+`López and Temme (2013) <https://arxiv.org/abs/1306.2046>`_ that converges in
+these regions. In addition, degenerate cases with one or more of ``a``, ``b``,
+and/or ``c`` a non-positive integer are now handled in a manner consistent with
+`mpmath's hyp2f1 implementation <https://mpmath.org/doc/current/functions/hypergeometric.html>`_,
+which fixes `#7340 <https://github.com/scipy/scipy/issues/7340>`_. These fixes
+were made as part of an effort to rewrite the Fortran 77 implementation of
+hyp2f1 in Cython piece by piece. This rewriting is now roughly 50% complete.
 
 `scipy.stats` improvements
 ==========================
 
-Hypothesis Tests
-----------------
+`scipy.stats.qmc.LatinHypercube` introduces two new optional keyword-only
+arguments, ``optimization`` and ``strength``. ``optimization`` is either
+``None`` or ``random-cd``. In the latter, random permutations are performed to
+improve the centered discrepancy. ``strength`` is either 1 or 2. 1 corresponds
+to the classical LHS while 2 has better sub-projection properties. This
+construction is referred to as an orthogonal array based LHS of strength 2.
+In both cases, the output is still a LHS.
+
+`scipy.stats.qmc.Halton` is faster as the underlying Van der Corput sequence
+was ported to Cython.
 
+The ``alternative`` parameter was added to the ``kendalltau`` and ``somersd``
+functions to allow one-sided hypothesis testing. Similarly, the masked
+versions of ``skewtest``, ``kurtosistest``, ``ttest_1samp``, ``ttest_ind``,
+and ``ttest_rel`` now also have an ``alternative`` parameter.
 
-Sample statistics
------------------
+Add `scipy.stats.gzscore` to calculate the geometrical z score.
 
+Random variate generators to sample from arbitrary univariate non-uniform
+continuous and discrete distributions have been added to the new
+scipy.stats.sampling submodule. Implementations of a C library
+`UNU.RAN <http://statmath.wu.ac.at/software/unuran/>`_ are used for
+performance. The generators added are:
 
-Statistical Distributions
--------------------------
+- TransformedDensityRejection
+- DiscreteAliasUrn
+- NumericalInversePolynomial
+- DiscreteGuideTable
+- SimpleRatioUniforms
 
+The ``binned_statistic`` set of functions now have improved performance for
+the ``std``, ``min``, ``max``, and ``median`` statistic calculations.
 
-Other
------
+``somersd`` and ``_tau_b`` now have faster Pythran-based implementations.
 
+Some general efficiency improvements to handling of ``nan`` values in
+several ``stats`` functions.
 
+Added the Tukey-Kramer test as `scipy.stats.tukey_hsd`.
 
+Improved performance of `scipy.stats.argus.rvs`.
+
+Added the parameter ``keepdims`` to `scipy.stats.variation` and prevent the
+undesirable return of a masked array from the function in some cases.
 
 *******************
 Deprecated features
 *******************
 
-`scipy.linalg` deprecations
-===========================
+Namespaces that were private but happened to miss underscores have been
+deprecated. These include:
```

Comment:
This needs an update. I'll write one. I think this is important, and perhaps should be added to the highlights for the release.

Reply:
Pushed a commit.